subprojects/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/tasks/ValidateTaskPropertiesIntegrationTest.groovy

```diff
@@ -387,6 +390,59 @@ class ValidateTaskPropertiesIntegrationTest extends AbstractIntegrationSpec {
         file("build/reports/task-properties/report.txt").text == ""
     }
 
+
+    @Unroll
+    def "report setters for mutable type #type"() {
+        @Language("JAVA") myTask = """
```

Comment:
`@Language("JAVA")` renders it in Idea like:
![image](https://user-images.githubusercontent.com/1841944/56966322-28146080-6b5f-11e9-97b9-db5ce0b7efd3.png)